scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/DBJobDataParameters.java

```diff
@@ -0,0 +1,67 @@
+package org.ow2.proactive.scheduler.core.db;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.ow2.proactive.db.SortParameter;
+import org.ow2.proactive.scheduler.common.JobSortParameter;
+import org.ow2.proactive.scheduler.common.job.JobStatus;
+
+/**
+ * Utility container to pass multiple parameters for jobs requests.
+ *
+ */
+public class DBJobDataParameters {
+    private int offset;
+    private int limit;
+    private String user;
+    private boolean pending;
+    private boolean running;
+    private boolean finished;
+    private List<SortParameter<JobSortParameter>> sortParameters;
+
+    DBJobDataParameters(int offset, int limit, String user,
+            boolean pending, boolean running, boolean finished,
+            List<SortParameter<JobSortParameter>> sortParameters) {
+        this.offset = offset;
+        this.limit = limit;
+        this.user = user;
+        this.pending = pending;
+        this.running = running;
+        this.finished = finished;
+        this.sortParameters = sortParameters;
+    }
+
+    public int getOffset() {
+        return offset;
+    }
+
+    public int getLimit() {
+        return limit;
+    }
+
+    public String getUser() {
+        return user;
+    }
+
+    public List<SortParameter<JobSortParameter>> getSortParameters() {
+        return sortParameters;
+    }
+
+    public Set<JobStatus> getStatuses() {
+
```

Comment:
I would suggest to define `status` variable as an instance variable and initialize it in the constructor. This way, if a someone call `getStatuses` twice or more it will not pay the price of executing the logic multiple times.